compare/README.md

```diff
@@ -40,6 +41,9 @@ More precisely, the test program (`compare`) would scan the input directory and
 5) fuzzy compare the rendering result with the corresponding png file;
 6) save the rendering result in the output directory if requested.
 
+Alternatively, image files can be placed into separate image directories, specified by `-t` command line switch.
```

Comment:
Nits: “Alternatively” imho sounds as if images would *no* longer be loaded from the input dir when `-t` is passed, which is not the case. “Additionaly” might be better.
Also I think just using Unicode `–` or just ASCII `-` instead of HTML's `&ndash;` would be more readable, but `&ndash;` is already used before this PR.

Reply:
(I agree on `&ndash;`: Markdown is meant to be readable as plain text. But it’s understandable enough, and especially if it’s already there, oh well.)